ripe/atlas/cousteau/source.py

```diff
@@ -127,6 +165,22 @@ def set_type(self, value):
     doc_type = "Defines the type of probe's source."
     type = property(get_type, set_type, doc=doc_type)
 
+    # tags attribute
+    def get_tags(self):
+        """Getter for tags attribute"""
+        return self._tags
+
+    def set_tags(self, value):
+        """Setter for tags attribute"""
+        super(AtlasChangeSource, self).set_tags(value)
+        if self.tags:
+            log = "Tag-based filtering can't be used when changing " \
+                  "participants probes for a measurement."
+            raise MalFormattedSource(log)
```

Comment:
Not sure if that's true. I believe you are allowed to use tags when action == "add". So I think same logic as the set_type should be applied. I will double check our internal v2  API code to see if we allow it for sure.